src/pim-types.ts

```diff
@@ -230,6 +230,10 @@ export class TaskType extends EventType {
     return this.dueDate && moment(this.dueDate.toJSDate()).isSameOrBefore(moment(), 'day');
   }
 
+  get overdue() {
+    return !!this.dueDate && moment(this.dueDate.toJSDate()).isBefore(moment(), 'day');
```

Comment:
This function is wrong. :)

There are two cases here:
1. `dueDate` is a date (rather than datetime) - then this is correct.
2. 'dueDate` is a datetime - then this is wrong, and you should just compare the date as is...

Reply:
ah yes I didn't think of that second case